pkg/util/tracing/tracer_span.go

```diff
@@ -329,6 +329,47 @@ func (r Recording) FindLogMessage(pattern string) (string, bool) {
 	return "", false
 }
 
+// CountRoundTrips returns the number of round trips found in the
```

Comment:
I don't think this package should really think about "round trips" -- its just spans here! It might make sense to pull this logic elsewhere, but still operate on the `Recording` object.